saleor/site/models.py

```diff
@@ -57,6 +82,27 @@ class Meta:
     def __str__(self):
         return self.site.name
 
+    @property
+    def default_from_email(self) -> Optional[str]:
+        sender_name: Optional[str] = self.default_mail_sender_name or ""
+        sender_address: Optional[str] = self.default_mail_sender_address
+
+        if not sender_address:
+            sender_address = settings.DEFAULT_FROM_EMAIL
+
+            if not sender_address:
+                return None
```

Comment:
Since this configuration is becoming dynamic, I think we should throw a warning when this property returns `None`. I'm even wondering if it shouldn't be thrown right away if `settings.DEFAULT_FROM_EMAIL` is used here. Site settings should be configured as early as possible.

Reply:
We are currently using `console:///` by default. Thus users would get an error when trying saleor. Maybe we could add the default `demo@example.com` in settings when `console:///` is used?